server/engine/tests/test_scale_osim.py

```diff
@@ -0,0 +1,37 @@
+import shutil
+import unittest
+from kinematics_pass.subject import Subject
+from kinematics_pass.trial import TrialSegment, Trial
+from typing import Dict, List, Any
+import os
+import nimblephysics as nimble
+from inspect import getsourcefile
+from utils.scale_osim import scale_osim
+
+TESTS_PATH = os.path.dirname(getsourcefile(lambda:0))
+DATA_PATH = os.path.join(TESTS_PATH, '..', '..', 'data')
+TEST_DATA_PATH = os.path.join(TESTS_PATH, '..', 'test_data')
+
+def reset_test_data(name: str):
+    original_path = os.path.join(TEST_DATA_PATH, f'{name}_original')
+    live_path = os.path.join(TEST_DATA_PATH, name)
+    if os.path.exists(live_path):
+        shutil.rmtree(live_path)
+    shutil.copytree(original_path, live_path)
```

Comment:
I noticed this was also used in `test_subject.py` and `test_trial.py`. If more testing code is reused, it could make sense to move them to a shared utility file/folder.

Reply:
Grouping the test files has made this largely go away